packages/react-internal/src/components/Image/Image.types.ts

```diff
@@ -0,0 +1,256 @@
+import * as React from 'react';
```

Comment:
These changes should all be re-applied under `packages/react/src/components/Image`. I was hoping that merging with master would automatically pick that up, but apparently not. :(

You should also delete `packages/react-internal/etc/react-internal.api.md`.